src/dash.ts

```diff
@@ -8,9 +8,9 @@ export class Dash {
       {
         darwin: 'open -g',
         linux: 'zeal',
-        // start Zeal before we pass it a URI so it works whether Zeal is running in background or not.
-        // Why is this needed? I have no idea, but I think Qt has to start up a listener on cmd before it will work.
-        win32: 'start dash-plugin:// && start',
+        // Same technique as Silverlake Software's "Search Docsets" extension,
+        // which is written by Velocity's developer and is tested to work with current Velocity on W10.
+        win32: 'cmd.exe /c start "" ',
```

Comment:
@salembeats have we tested this with Zeal?

Reply:
I haven't wanted to try it out on Zeal on my main machine because installing Zeal breaks Velocity's integration for Dash URLs. It should work, though. I'll test it on a VM in a minute.

Reply:
Yep, did some quick testing in VM and Zeal doesn't seem to mind the change:

![Code/Zeal VM Screenshot](https://i.imgur.com/PwTbfED.png)